### DISCUSSION TITLE
Caddy + No-Ip + Dns Challenge

### DISCUSSION THREAD

**Discussion by flauzf** [indomus/forum#2581] [GENERALE - Discussioni generiche]
Ciao, 
seguendo le vostre guide ho installato HA core su docker su minipc - 
Sono riuscito a configurare quanto mi occorreva, poi con calma e tempo permettendo studierò il resto.
Ho installto Caddy e riesco a raggiungere HA dall'esterno via https su una porta dedicata (non 443) che ho dovuto comunque aprire sul firewall vs HA.
Visto che pago già per altri motivi l'abbonamento a no-ip, mi piacerebbe intergare la DNS challenge. 
Su git-hub caddy-dns non trovo però il modulo no-ip. Qualche esperienza in merito? Nel frattempo mi leggo la doc su caddyserver.com
grazie mille
F.

**Comment by snakuzzo** [comment]
Ciao.
L'unica cosa è seguire quanto scritto [QUI](https://caddy.community/t/how-to-use-dns-provider-modules-in-caddy-2/8148#if-you-do-not-find-your-dns-provider-4).

**Reply by flauzf** [reply]
Grazie per la dritta.
